connector/authproxy/authproxy.go

```diff
@@ -69,7 +70,8 @@ func (m *callback) HandleCallback(s connector.Scopes, r *http.Request) (connecto
 	groups := m.groups
 	headerGroup := r.Header.Get(m.groupHeader)
 	if headerGroup != "" {
-		groups = append(groups, headerGroup)
+		trimHeaderGroup := strings.Replace(headerGroup, " ", "", -1)
```

Comment:
I wonder whether a group name can contain whitespaces? Like
```
group 1, group 2
```

Reply:
I updated the connector to support that cases 😄